src/strapp/http/client.py

```diff
@@ -88,6 +88,8 @@ def make_request(
             (
                 requests.exceptions.HTTPError,
                 requests.exceptions.ConnectionError,
+                requests.exceptions.ConnectTimeout,
+                requests.exceptions.ReadTimeout,
```

Comment:
ReadTimeout typically indicates that we need to increase the per-request timeout, iirc. Like making a request which returns 10k items might legitimately take longer than the default 20s or whatever. Sometimes retrying them can help, but i think for that particular error kind we're better off targeting the timeout on the request or backing off at the call site instead.

Reply:
ah makes sense